.github/workflows/publish-theia-ide-img.yml

```diff
@@ -57,3 +60,14 @@ jobs:
             ghcr.io/${{ github.repository }}/theia-ide:${{ github.event.inputs.tag }}
             ghcr.io/${{ github.repository }}/theia-ide:latest
           platforms: linux/amd64,linux/arm64
+
+      - name: Build and push Browser Base IDE Docker image
+        uses: docker/build-push-action@48aba3b46d1b1fec4febb7c5d0c644b249a11355 # v6.10.0
+        with:
+          context: .
+          file: images/base-ide/Dockerfile
+          push: true
+          tags: |
+            ghcr.io/${{ github.repository }}/base-ide:${{ github.event.inputs.tag }}
+            ghcr.io/${{ github.repository }}/base-ide:latest
+          platforms: linux/amd64,linux/arm64
```

Comment:
This should be build before the theia-ide image above, as we want to use the new base image for the theia-ide image.